src/components/elements/FilterBox.js

```diff
@@ -0,0 +1,53 @@
+import { useRef } from 'react';
+import classes from "./FilterBox.module.css";
+
+function FilterBox(props) {
+
+    const filterRef = useRef();
+
+  function filterHandler(filterMethod) {
+
+    const postBody = filterRef.current.value;
+
+    if (filterRef.length == 0) {
+        postBody = " ";
+    }
+
+    fetch(`http://localhost:8080/users/filter/${filterMethod}`, {
```

Comment:
Can you create a host variable?
The reason is when you have different env configs (dev/qa/uat/prod) you can read the property from a env-specific config file instead of using a hard-coded destination

Reply:
What would implementing a host variable look like? My guess is: it'd be something like a separate file containing a URL that all the fetch() commands call from?

Reply:
By 'host variable' I just meant a normal variable here to acknowledge that this would change depending on the environment.

See https://create-react-app.dev/docs/adding-custom-environment-variables to learn about env vars.
I suggest you skip it for now since you want to see your app working e2e before moving on to the grad project